neqo-transport/src/connection.rs

```diff
@@ -2218,4 +2300,25 @@ mod tests {
             _ => false,
         });
     }
+
+    #[test]
+    fn test_idle_timeout() {
```

Comment:
Do we need to add `test_` to the name of these functions?  I haven't been and it's nice having fewer characters to type when filtering tests.

Reply:
yeah, no we don't. Probably my doing so many thus named exist in the tree. We can omit. I'll leave doing that en masse for later, but update the test this PR adds, at least.